src/components/manage/Blocks/Accordion/Edit.jsx

```diff
@@ -182,8 +212,6 @@ const Edit = (props) => {
                     },
                   },
                 });
-              } else {
-                onChangeField(id, value);
```

Comment:
@sneridagh Not sure why did you remove this. I think this is required when using together with volto-metadata-block

Reply:
See https://github.com/eea/volto-accordion-block/pull/22/files

Reply:
Oh I didn't! I will chech the pr again